Modules/Models/Configuration.swift

```diff
@@ -345,66 +345,71 @@ public extension Dictionary where Key == String, Value == [String: String] {
-        "DE": "DEUTSCHLAND",
-        "IE": "IRLAND",
-        "LV": "LETTLAND",
-        "NL": "NIEDERLANDE",
-        "PL": "POLEN",
-        "CZ": "TSCHECHISCHE REPUBLIK",
-        "ES": "SPANIEN"
-      ],
-      "en": [
-        "AT": "AUSTRIA",
-        "DK": "DENMARK",
-        "EE": "ESTONIA",
-        "DE": "GERMANY",
-        "IE": "IRELAND",
-        "LV": "LATVIA",
-        "NL": "NETHERLANDS",
-        "PL": "POLAND",
-        "CZ": "CZECH REPUBLIC",
-        "ES": "SPAIN"
-      ],
-      "fr": [
-        "AT": "AUTRICHE",
-        "DK": "DANEMARK",
-        "EE": "ESTONIE",
-        "DE": "ALLEMAGNE",
-        "IE": "IRLANDE",
-        "LV": "LETTONIE",
-        "NL": "PAYS-BAS",
-        "PL": "POLOGNE",
-        "CZ": "RÉPUBLIQUE TCHÈQUE",
-        "ES": "ESPAGNE"
-      ],
-      "es": [
-        "AT": "AUSTRIA",
-        "DK": "DINAMARCA",
-        "EE": "ESTONIA",
-        "DE": "ALEMANIA",
-        "IE": "IRLANDA",
-        "LV": "LETONIA",
-        "NL": "PAÍSES BAJOS",
-        "PL": "POLONIA",
-        "CZ": "REPÚBLICA CHECA",
-        "ES": "ESPAÑA"
-      ]
+        "it": [
+            "AT": "AUSTRIA",
+            "HR": "CROAZIA",
+            "DK": "DANIMARCA",
+            "EE": "ESTONIA",
+            "DE": "GERMANIA",
+            "IE": "IRLANDA",
+            "LV": "LETTONIA",
+            "NL": "OLANDA",
+            "PL": "POLONIA",
+            "CZ": "REPUBBLICA CECA",
+            "ES": "SPAGNA"
+            ],
+        "de": [
+            "AT": "ÖSTERREICH",
+            "HR": "KROATIEN",
+            "DK": "DÄNEMARK",
+            "EE": "ESTONIA",
+            "DE": "DEUTSCHLAND",
+            "IE": "IRLAND",
+            "LV": "LETTLAND",
+            "NL": "NIEDERLANDE",
+            "PL": "POLEN",
+            "CZ": "TSCHECHISCHE REPUBLIK",
+            "ES": "SPANIEN"
+            ],
+        "en": [
+            "AT": "AUSTRIA",
+            "HR": "CROATIA",
+            "DK": "DENMARK",
+            "EE": "ESTONIA",
+            "DE": "GERMANY",
+            "IE": "IRELAND",
+            "LV": "LATVIA",
+            "NL": "NETHERLANDS",
+            "PL": "POLAND",
+            "CZ": "CZECH REPUBLIC",
+            "ES": "SPAIN"
+            ],
+        "fr": [
+            "AT": "AUTRICHE",
+            "HR": "CROATIE",
+            "DK": "DANEMARK",
+            "EE": "ESTONIE",
+            "DE": "ALLEMAGNE",
+            "IE": "IRLANDE",
+            "LV": "LETTONIE",
+            "NL": "PAYS-BAS",
+            "PL": "POLOGNE",
+            "CZ": "RÉPUBLIQUE TCHÈQUE",
+            "ES": "ESPAGNE"
+            ],
+        "es": [
+            "AT": "AUSTRIA",
+            "HR": "CROACIA",
+            "DK": "DINAMARCA",
+            "EE": "ESTONIA",
+            "DE": "ALEMANIA",
+            "IE": "IRLANDA",
+            "LV": "LETONIA",
+            "NL": "PAÍSES BAJOS",
+            "PL": "POLONIA",
+            "CZ": "REPÚBLICA CHECA",
+            "ES": "ESPAÑA"
+            ]
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: Modules/Models/Configuration.swift;
  Line: 412;
-->

- :warning: swiftlint (0.39.2): Array and dictionary literal end should have the same indentation as the line that started it. Expected 8, got 12. (literal_expression_end_indentation)